test/unittest/tests/test_extent_list.cpp

```diff
@@ -0,0 +1,425 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+#include <memory>
+
+#include <catch2/catch.hpp>
+
+#include "wt_internal.h"
+
+struct ExtentWrapper {
+    ExtentWrapper(WT_EXT *raw) : _raw(raw) {}
+
+    ~ExtentWrapper()
+    {
+        free(_raw);
+    }
+
+    WT_EXT *_raw;
+};
+
+struct ExtentListWrapper {
+    ExtentListWrapper() {}
```

Comment:
Do we need the constructor and the destructor if there is no specific definition?
Instead you could use:
```
ExtentListWrapper() = default;
~ExtentListWrapper() = default;
```
Same applies for the `SizeListWrapper` class.

Reply:
You're totally right, fixed in 117f0bc.